course_catalog/tasks.py

```diff
@@ -6,6 +6,7 @@
 
 from course_catalog.settings import EDX_API_URL
 from course_catalog.tasks_helpers import get_access_token, parse_mitx_json_data
+from course_catalog.utils import log
```

Comment:
For consistency with the `open-discussions` repo style, can you do this instead:

- Add `import logging` above `import requests`
then:
```python
from course_catalog.tasks_helpers import get_access_token, parse_mitx_json_data


log = logging.getLogger(__name__)

```